src/main/java/com/jalasoft/webservice/model/VideoConverterCriteria.java

```diff
@@ -0,0 +1,107 @@
+package com.jalasoft.webservice.model;
+import java.io.File;
+
+public class VideoConverterCriteria extends Criteria {
+
+
+     private File target;
+     private String setCodec;
+     private int setBitRate;
+     private int setChannels;
+     private int setSamplingRate;
+     private int setFrameRate;
+     private int setSizeX;
+     private int setSizeY;
+     private String setFormat;
+     private String setAudioAttributes;
+     private String setVideoAttributes;
+
+    public File getTarget() {
```

Comment:
This class does not have documetation.

Reply:
Thank you for realizing about it because now I am doing the necessary correction.